kratos/processes/calculate_octree_signed_distance_to_3d_skin_process.h

```diff
@@ -459,9 +458,10 @@ class DistanceSpatialContainersConfigure
           ModelPart::NodesContainerType::ContainerType& nodes = mrBodyModelPart.NodesArray();
           int nodes_size = nodes.size();
           // first of all we reset the node distance to 1.00 which is the maximum distnace in our normalized space.
-#pragma omp parallel for firstprivate(nodes_size)
-          for(int i = 0 ; i < nodes_size ; i++)
-              nodes[i]->GetSolutionStepValue(DISTANCE) = 1.00;
+
+        IndexPartition<std::size_t>(nodes_size).for_each([&](std::size_t Index){
+            nodes[Index]->GetSolutionStepValue(DISTANCE) = 1.00;
```

Comment:
variableUtils

Reply:
Yes. Actually I did that before in this [commit](https://github.com/KratosMultiphysics/Kratos/pull/8507/commits/0e3050cc5845ce5f5a48aeca0745b1b14f52ce8a) . But then removed as I was not sure about the errors, because this file is not included in the compilation. 
Anyways I did it again. Thanks :)

Reply:
ah that was that file
since probably it will be removed you can leave it as is for the moment I would say